### PR TITLE
Adding tests

### PR DESCRIPTION
**Pull request by dhoehna**
Adding tests for ChrW.  The tests for AscW existed already.

### CODE REVIEW

**Comment by danmosemsft** [comment]
@dhoehna thanks. Right now we don't run tests against desktop during CI, so could you please make sure they pass. From the root, you'll probably need to do 
```
build.cmd -framework:netfx
build-tests.cmd -framework:netfx
```
after that you can run against this project by
`msbuild /t:buildandtest /p:targetgroup=netfx`

File: src/Microsoft.VisualBasic/tests/StringsTests.cs [hunk 1]
```diff
@@ -20,5 +20,24 @@ public void AscWTest()
...
+
+        [Theory]
+        [InlineData(97)]
+        [InlineData(65)]
+        [InlineData(65535)]
+        [InlineData(-32768)]
+        public void ChrW_CharCodeInRange(int charCode)
+        {
+            char result = Strings.ChrW(charCode);
+            Assert.Equal(System.Convert.ToChar(charCode & 0XFFFF), result);
```
Comment (stephentoub) [inline src/Microsoft.VisualBasic/tests/StringsTests.cs]: Nit: is the "System." necessary for disambiguation, or could it be removed?

**Comment by dhoehna** [comment]
@danmosemsft  I've tried running build-tests.cmd -framework:netfx two times.  Each time the tests get stuck on a particular test (I don't remember which ones).  Once the build tests get stuck, I have to close the command window.

I also keep getting a popup about the "RemoteExecutorConsoleApp" has stopped working.  What is the error about?

I am running the tests a third time to see what test the developer console gets stuck on.

**Comment by danmosemsft** [comment]
@dhoehna the desktop runs are not fully stabilized. +@safern. However to the extent it worked, enough may have happened that `msbuild /t:buildandtest /p:targetgroup=netfx` works in your test folder...

**Comment by danmosemsft** [comment]
@safern

**Comment by dhoehna** [comment]
This time the run ran into an error that was so long I can't see the top in my command window.  Also, I have 12 instances of that pop-up error.

**Comment by danmosemsft** [comment]
@dhoehna the only other thing I can suggest is `git pull` on master, change to your branch, `git rebase master`, then `clean -all` from the root, then repeat :
```
build.cmd -framework:netfx
build-tests.cmd -framework:netfx
```
ignoring any issues, then
`msbuild /t:rebuildandtest /p:targetgroup=netfx`
on the VB test project.

As there was a recent tools update that maybe messed things up unless you clean like this.

if that doesn't work then @safern can advise

**Comment by safern** [comment]
There is no need to do `build-tests.cmd -framework:netfx` first of all that will build all the tests projects across the repo an execute them. As we are not stable yet to do that, it would bring a lot of errors. There is also a way to build all the test projects without executing the tests which is ```build-tests.cmd -framework:netfx -skiptests``` but I think what you want is only to run the tests under the VB project which I will recommend is to do what @danmosemsft suggested skipping the `build-tests.cmd -framework:netfx` step. 

```
clean.cmd -all
build.cmd -framework:netfx
msbuild src\Microsoft.VisualBasic\tests\Microsoft.VisualBasic.Tests.csproj /t:rebuildandtest /p:targetgroup=netfx
```

Right now there is only one test and it passes, so you shouldn't see any failures.

**Comment by dhoehna** [comment]
Pushed some a new commit.  I tested the commit before I pushed.  The tests ran successfully to completion.

**Pull request merged**